app/ui/screeningsTableStartpage.tsx

```diff
@@ -13,20 +14,27 @@ import { DatePicker } from '@mui/x-date-pickers/DatePicker';
 import dayjs, { Dayjs } from 'dayjs';
 import 'dayjs/locale/de';
 import { Screening } from '../lib/definitions';
+import { useSearchParams } from 'next/navigation';
```

Comment:
You're importing useSearchParams and later assigning it's function to a constant, but you never use it so I think you should take it away

Reply:
You are right, ive removed it now.